src/components/home/JoinText.tsx

```diff
@@ -0,0 +1,9 @@
+const JoinText = () => {
+  return (
+    <div className="flex w-full justify-center">
+      <p className="w-[70%] text-[4vw]">WHY JOIN?</p>
```

Comment:
```suggestion
      <p className="text-[4vw]">WHY JOIN?</p>
```
instead of w-[70%], you can just use w-fit to restrict the height. Also, pull from dev and use the font I just added, it should be font-seasons